HomeWorks/java/lesson3/Task1HW3.java

```diff
@@ -0,0 +1,20 @@
+package lesson3;
+
+public class Task1HW3 {
+    public static void main(String[] args) {
+        String shortString = "word";
+        String longString = "helloWorld";
+        String theLongestString = "GoodMorningWorld";
+        int cont1 = shortString.length();
+        int cont2 = longString.length();
+        int cont3 = theLongestString.length();
+        if (cont1 > cont2) {
```

Comment:
it's better to locate it in a separate method:
 static String wordGreatLength(String word1, String word2, String word3) {
        int wordlenght1 = word1.length();
        int wordlenght2 = word2.length();
        int wordlenght3 = word3.length();

        if ((wordlenght1 > wordlenght2) && (wordlenght1 > wordlenght3)) {
            return word1;
        } else if ((wordlenght2 > wordlenght1) && (wordlenght2 > wordlenght3)) {
            return word2;
        } else {
            return word3;
        }
    }